deploy/deployment.yml

```
@@ -0,0 +1,92 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: ktr
+  name: token-refresher
+  namespace: catalog
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: token-refresher
+  template:
+    metadata:
+      labels:
+        app: token-refresher
+    spec:
+      containers:
+      - name: refresher
+        env:
+        - name: KTR_SECRETNAME
+          value: "bearer-creds"
+        - name: KTR_SECRETNAMESPACE
+          valueFrom:
+            fieldRef:
+              apiVersion: v1
+              fieldPath: metadata.namespace
+        - name: KTR_SECRETKEY
+          value: "token"
+        - name: KTR_OIDC_TOKENURL
+          value: "https://auth.vshn.net/token"
+        - name: KTR_OIDC_CLIENTID
+          valueFrom:
+            secretKeyRef:
+              name: ktr
+              key: id
+        - name: KTR_OIDC_CLIENTSECRET
+          valueFrom:
+            secretKeyRef:
+              name: ktr
+              key: secret
+        image: docker.io/vshn/kube-token-refresher:v1.0.0
```

Due to Docker's Pull limit we stopped using Docker.io as the default registry.
Additionally, to keep our release process slim, we should not pin the version here, otherwise we have to update the tag before each release.
```suggestion
        image: quay.io/vshn/kube-token-refresher:latest
```